File: mashup-recruit/src/main/java/kr/mashup/branding/ui/team/TeamResponse.java
package kr.mashup.branding.ui.team;

import kr.mashup.branding.domain.team.Team;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class TeamResponse {
    private Long teamId;
    private String name;

    public static TeamResponse from(Team team){
        return new TeamResponse(team.getTeamId(), team.getName());
    }
}

File: mashup-recruit/src/main/java/kr/mashup/branding/ui/team/TeamResponse.kt
package kr.mashup.branding.ui.team

import kr.mashup.branding.domain.team.Team

data class TeamResponse(
    val teamId: Long,
    val name: String
) {

    companion object {
        @JvmStatic
        fun from(team: Team): TeamResponse {
            return TeamResponse(team.teamId, team.name)
        }
    }
}